my-code/src/components/LikeButton/index.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Container, Button } from './styles';
+import iconHeartWhite from './assets/icon-heart-white.svg';
+import iconHeartGrey from './assets/icon-heart-grey.svg';
+import iconHeartFull from './assets/icon-heart-full.svg';
+
+export default function LikeButton({ imdbID, showText, alwaysVisible }) {
+  const [liked, setLiked] = useState(false);
+  const dispatch = useDispatch();
+
+  // Redux state: get the favourites list.
+  const likes = useSelector(state => state.likesReducer);
```

Comment:
It is a great idea to put all selectors on the reducer file, then if someday the "model" of reducer has changed, then it will be easier to update all selectors. Got it?

Reply:
Got it and did it.